Plugin/clScrolledPanel.cpp

```diff
@@ -384,7 +384,7 @@ wxFont clScrolledPanel::GetDefaultFont()
     f.SetFractionalPointSize(pointSize);
 #elif defined(__WXMSW__)
     // wxMSW
-    float pointSize = f.GetFractionalPointSize() * 1.2;
+    float pointSize = f.GetFractionalPointSize();
     f.SetFractionalPointSize(pointSize);
```

Comment:
So whole `__WXMSW__` block can be removed

Reply:
Something like this, i guess.

Reply:
No need to set same value for other, simply
```
wxFont clScrolledPanel::GetDefaultFont()
{
    wxFont f = wxSystemSettings::GetFont(wxSYS_DEFAULT_GUI_FONT);
#if defined(__WXMAC__)
    float pointSize = f.GetFractionalPointSize() * 1.2;
    f.SetFractionalPointSize(pointSize);
#endif
    return f;
}
```
or
```
wxFont clScrolledPanel::GetDefaultFont()
{
    wxFont f = wxSystemSettings::GetFont(wxSYS_DEFAULT_GUI_FONT);
#if defined(__WXMAC__)
    f.SetFractionalPointSize(1.2 * f.GetFractionalPointSize());
#endif
    return f;
}
```

Reply:
Yes, really, I am setting the same value back.